client/components/Editor/plugins/ids.js

```diff
@@ -0,0 +1,93 @@
+import { Plugin } from 'prosemirror-state';
```

Comment:
This is a lightly retouched version of Travis' plugin for this here: https://github.com/pubpub/pubpub/blob/21391827664bb8ab15ea4aebedb3ddd37d8d1d62/client/components/Editor/plugins/hashes.js